Include/compile.h

```diff
@@ -64,6 +64,14 @@ PyAPI_FUNC(PyCodeObject *) PyAST_CompileObject(
     PyCompilerFlags *flags,
     int optimize,
     PyArena *arena);
+PyAPI_FUNC(PyCodeObject *) _PyAST_Compile(
+    struct _mod *mod,
+    PyObject *filename,
+    PyCompilerFlags *flags,
+    int optimization_level,
+    PyArena *arena,
+    int noopt);
```

Comment:
optimization_level=0 still means having optimizations enabled. I cannot change to keep the backward compatibility.

Do you think that it would be better to add a new ``PyCF_xxx`` flag to avoid adding this new function?